### PR TITLE
Fix/post detail and like

### PR DESCRIPTION
**Pull request by Docker-J**
Update detail post view with new heart like component and comment showing component
also fix for likes

### CODE REVIEW

File: src/api/user_api.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
 import axios from "axios";
 
 //TODO: cite this
```
Comment (laingkob) [inline src/api/user_api.js:3]: Lol we should do this before we forget

File: src/components/ListItems/post-list.js [hunk 2]
```diff
@@ -16,25 +16,29 @@ function PostList({ user_list }) {
 
   function checkLiked(item) {
     for (var i = 0; i < liked.length; i++) {
-      if (liked[i].object === item.id) {
```
Comment (laingkob) [inline src/components/ListItems/post-list.js:19]: Do we not need triple = ?

**Review by laingkob — CHANGES_REQUESTED** [review]
Could you please add the citation from the docs? I know it is from a while ago so we should add it before we forget, in case the prof decides to push it as plagiarism (which would be dumb but I just want to make sure)

Sorry I totally forgot about the post detail page when I made the like component, glad you thought to put it in there. Thanks for cleaning up the state stuff in that too.

File: src/components/ListItems/post-list.js (hunk 2 above)
Comment (Docker-J) [reply src/components/ListItems/post-list.js:19]: true. I was testing around and forget to redo this

**Pull request merged**